boilerplate/app/screens/DemoComponentsScreen/DemoComponentsScreen.tsx

```diff
@@ -44,6 +45,16 @@ export function DemoComponentsScreen(_props: DemoTabScreenProps<"DemoComponents"
     toggleDrawer()
   }
 
+  const scrollToIndexFailed = (info: { index: number; highestMeasuredFrameIndex: number; averageItemLength: number;}) => {
+    listRef.current?.getScrollResponder()?.scrollToEnd()
+    timeout.current = setTimeout(() => listRef.current?.scrollToLocation({animated: true, itemIndex: info.index, sectionIndex: 0}), 50) 
```

Comment:
Whenever the scroll to index fails, we scroll to end of list to render the rest of the items, then scroll to the index location.